ui/oopsyraidsy/data/00-misc/general.ts

```diff
@@ -151,35 +151,35 @@ const triggerSet: OopsyTriggerSet<Data> = {
       type: 'LosesEffect',
       netRegex: NetRegexes.losesEffect({ effectId: '94' }),
       run: (data, matches) => {
-        (data.raiseLostTracker ??= {})[matches.targetId] = matches.timestamp;
+        (data.lastRaisedLostTime ??= {})[matches.targetId] = matches.timestamp;
       },
     },
     {
       id: 'General Double Raise',
       type: 'GainsEffect',
       netRegex: NetRegexes.gainsEffect({ effectId: '94' }),
       mistake: (data, matches) => {
-        data.raiseLostTracker ??= {};
+        data.lastRaisedLostTime ??= {};
         data.originalRaiser ??= {};
-        const lastRaiser = data.originalRaiser[matches.targetId];
+        const originalRaiser = data.originalRaiser[matches.targetId];
         // 30 and 26 lines having the same timestamp means effect was overwritten and the target is still dead
-        const overwrittenRaise = data.raiseLostTracker[matches.targetId] === matches.timestamp;
+        const overwrittenRaise = data.lastRaisedLostTime[matches.targetId] === matches.timestamp;
         // if that's not the case, target doesn't have a raise yet
         if (!overwrittenRaise) {
           data.originalRaiser[matches.targetId] = data.ShortName(matches.source);
           return;
         }
         // otherwise, report overwritten raise
-        if (lastRaiser !== undefined) {
-          delete data.raiseLostTracker[matches.targetId];
+        if (originalRaiser !== undefined) {
+          delete data.lastRaisedLostTime[matches.targetId];
           return {
             type: 'warn',
             blame: matches.source,
             reportId: matches.sourceId,
             text: {
-              en: 'overwrote ' + lastRaiser + '\'s raise',
-              de: 'überschrieb ' + lastRaiser + '\'s Wiederbeleben',
-              ko: lastRaiser + '의 부활과 겹침',
+              en: 'overwrote ' + originalRaiser + '\'s raise',
+              de: 'überschrieb ' + originalRaiser + '\'s Wiederbeleben',
+              ko: originalRaiser + '의 부활과 겹침',
```

Comment:
```suggestion
              en: `overwrote ${originalRaiser}'s raise`,
              de: `überschrieb ${originalRaiser}'s Wiederbeleben`,
              ko: `${originalRaiser}의 부활과 겹침`,
```

and the same below too